tests/unit/progressbar/progressbar_core.js

```diff
@@ -1,28 +1,29 @@
 module( "progressbar: core" );
 
-test( "markup structure", function() {
-	expect( 5 );
-	var element = $( "#progressbar" ).progressbar();
-	ok( element.hasClass( "ui-progressbar" ), "main element is .ui-progressbar" );
-	ok( !element.hasClass( "ui-progressbar-indeterminate" ),
-		"main element is not .ui-progressbar-indeterminate" );
-	equal( element.children().length, 1, "main element contains one child" );
-	ok( element.children().eq( 0 ).hasClass( "ui-progressbar-value" ),
-		"child is .ui-progressbar-value" );
+test( "markup structure", function( assert ) {
+	expect( 6 );
+	var element = $( "#progressbar" ).progressbar(),
+		value = element.children().eq( 0 );
+
+	assert.hasClasses( element, "ui-progressbar ui-widget ui-widget-content" );
+	assert.lacksClasses( value, "ui-progressbar-complete" );
+
+	element.progressbar( "option", "value", 100 );
+	assert.lacksClasses( element, "ui-progressbar-indeterminate" );
+	assert.hasClasses( value, "ui-progressbar-complete ui-widget-header ui-progressbar-value" );
+	equal( element.children().length, 1, "Main element contains one child" );
 	equal( element.children().children().length, 0, "no overlay div" );
 });
 
-test( "markup structure - indeterminate", function() {
-	expect( 5 );
+test( "markup structure - indeterminate", function( assert ) {
+	expect( 4 );
 	var element = $( "#progressbar" ).progressbar({ value: false });
-	ok( element.hasClass( "ui-progressbar" ), "main element is .ui-progressbar" );
-	ok( element.hasClass( "ui-progressbar-indeterminate" ),
-		"main element is .ui-progressbar-indeterminate" );
-	equal( element.children().length, 1, "main element contains one child" );
-	ok( element.children().eq( 0 ).hasClass( "ui-progressbar-value" ),
-		"child is .ui-progressbar-value" );
+
+	assert.hasClasses( element, "ui-progressbar ui-progressbar-indeterminate" );
+	equal( element.children().length, 1, "Main element contains one child" );
+	assert.hasClasses( element.children().eq( 0 ), "ui-progressbar-value" );
```

Comment:
Still using multiple references to `element.children()` instead of using a var like the previous test.